plugins/module_utils/aci.py

```diff
@@ -1358,3 +1295,66 @@ def dump_json(self):
             if(output_path is not None):
                 with open(output_path, "a") as output_file:
                     json.dump([mo], output_file)
+
+    def api_call(self, method):
```

Comment:
why this new api_call function? request function already present, should that not be reworked?

Reply:
Since we are adding HTTP API connection plugin functionality, it has its own request method. In order to use that we need to make use of send_request().

fetch_url() is our current direct approach for making requests.

Reply:
Yes, request() can be modified to do what api_call() is doing now. 
request() is only used in one other function.